### ISSUE TITLE
PageBreakSource is missing from the Expressing Accessibility Metadata in the EPUB Package Document

### ISSUE THREAD

**Issue by gautierchomel** [w3c/publ-a11y#743]
The [Expressing Accessibility Metadata in the EPUB Package Document](https://w3c.github.io/publ-a11y/epub-a11y-meta-guide/1.0/draft/index.html) is missing information about the Page break source identification (`pageBreakSource`).

**Comment by mattgarrish** [comment]
I also raised this in https://github.com/w3c/publ-a11y/issues/726

**Comment by gautierchomel** [comment]
Independent of the display question, having the metadata informed sounds necessary in an academic context, at least. 

This one is about having it in the "Expressing metadata guide". We'll undoubtedly find use cases for this information in the Annotation exchange, by the way.
